src/modules/launcher/Wox.Plugin/Common/ShellLocalization.cs

```diff
@@ -39,12 +41,7 @@ public string GetLocalizedName(string path)
 
             shellItem.GetDisplayName(SIGDN.NORMALDISPLAY, out string filename);
 
-            if (!_localizationCache.ContainsKey(path.ToLowerInvariant()))
-            {
-                // The if condition is required to not get timing problems when called from an parallel execution.
-                // Without the check we will get "key exists" exceptions.
-                _localizationCache.Add(path.ToLowerInvariant(), filename);
-            }
+            _localizationCache.TryAdd(lowerInvariantPath, filename);
```

Comment:
Shouldn't it be the following?


```suggestion
            _ = _localizationCache.TryAdd(lowerInvariantPath, filename);
```

Reply:
Done.